extra_tests/snippets/builtin_format.py

```diff
@@ -29,3 +29,7 @@ def test_zero_padding():
 assert f'{4294967296:,}' == '4,294,967,296'
 assert 'F' == "{0:{base}}".format(15, base="X")
 assert f'{255:#X}' == "0XFF"
+assert f"{65:c}" == "A"
+assert f"{0x1f5a5:c}" == "🖥"
+assert_raises(ValueError, "{:+c}".format, 1, _msg="Sign not allowed with integer format specifier 'c'")
```

Comment:
This is added here because CPython doesn't have those tests, right?

Reply:
Similar test is in `test_types.py`.
https://github.com/RustPython/RustPython/blob/cb4bd51489188cb37010db26902b42b8948e7213/Lib/test/test_types.py#L360
But `test_int__format__` is docorated with `@unittest.expectedFailure`.
So I've added it in `builtin_format.py`.